src/pybind/mgr/dashboard/frontend/src/app/shared/components/card/card.component.html

```diff
@@ -1,7 +1,18 @@
-<div class="card shadow-sm flex-fill">
-  <h4 class="card-title mt-4 ms-4 mb-0">
-    {{ cardTitle }}
+<div class="card flex-fill"
```

Comment:
you can conditionally add shadow here so you can have shadow on the card you need.
```suggestion
<div class="card flex-fill"
        [ngClass]={"shadow": iNeedShadow}
```

Reply:
Done